pkg/broker/broker.go

```diff
@@ -27,6 +27,8 @@ type APISurface struct {
 const (
 	instanceIDVarKey = "instance_id"
```

Comment:
Could it be possible to add these constants to the `go-open-service-broker-client` and export them? WDYT?

Reply:
Yeah, I have a note elsewhere to export constants in the client, we should do that :)